cmd/edit/machinepool/nodepool_test.go

```diff
@@ -121,4 +121,28 @@ var _ = Describe("Nodepool", func() {
 			Expect(invoked).To(BeFalse())
 		})
 	})
+
+	Context("getNodePoolReplicas", func() {
```

Comment:
It's not related to the removal of version parameter, just to bump up the test coverage to pass the check.